Project/Library/LibraryViewController.swift

```diff
@@ -8,22 +8,63 @@
 
 import Foundation
 import UIKit
-class LibraryViewController: UIViewController {
+class LibraryViewController: UIViewController, UITableViewDelegate, UITableViewDataSource {
+    
+    private let _viewModel : BookViewModel = BookViewModel()
+    private let _view: BooksTableView = BooksTableView.loadFromNib()!
+    private var books : [Book] = []
+    
+    init() {
+        super.init(nibName:nil, bundle:nil)
+        tabBarItem =  UITabBarItem(title: "Library", image: UIImage(named:"ic_library.png"), selectedImage: UIImage(named: "ic_library"))
+        //loading the books from the source
+        _viewModel.loadBooks();
+        books = _viewModel.getBooks()
+        view.backgroundColor = Constants.backgroundColor
```

Comment:
We usually recommend creating extensions for each type. For example, after you create an `Extensions` folder, you can create a `UIColorExtension.swift` which expands the base class functionality.

```
extension UIColor {
    static var darkSpringGreen: UIColor {
        return UIColor(hex: "YOUR COLOR HERE")!
    }
}
```

Then you can just go `view.backgroundColor = .backgroundColor` (and you can skip the `UIColor` part for the right side, as Swift infers type)